tests/mitoai_ui_tests/mitoai.spec.ts

```diff
@@ -272,5 +272,10 @@ test.describe('Mito AI Chat', () => {
     await page.keyboard.type("@none_type_col_A");
     await expect(page.locator('.chat-dropdown-item-name').filter({ hasText: 'none_type_col_A' })).toBeVisible();
   });
+
+  test('CMD+E opens the AI chat', async ({ page }) => {
```

Comment:
Let's just update this test to make sure that the chat taskpane is closed to begin with. If it is, we can close it first and then open it with CMD + E